libcaf/libcaf/tag.py

```diff
@@ -0,0 +1,60 @@
+from pathlib import Path
+from .ref import HashRef, RefError, read_ref, write_ref
+
+from .constants import HASH_CHARSET, HASH_LENGTH
+
+class TagError(Exception):
+    """Base class for all tag-related errors."""
+
+class TagNotInTagsDirError(TagError):
```

Comment:
This is not really a good name for several reasons:
1. it's unusual and hard to remember and look up/find
2. it's long
3. it reveals implementation details that the consumer of this file (tag.py) does not and should know about
What's wrong with `TagNotFound` etc?